### PR TITLE
3 add and remove items from the list

### PR DESCRIPTION
**Pull request by pedrosilva410**

### CODE REVIEW

**Comment by 201flaviosilva** [comment]
Two notes before starting to review:
- Describe the PRs in the description, for example: post a picture of the state of the app, and describe what you've done, and anything else you think is relevant;
- The name of the commits must be more descriptive of what you did in that commit, not just "[file](https://github.com/pedrosilva410/todolist/pull/4/commits/0b42a7946cd9dcf8cb1dd69f2c7af5c9833e8860)". Check for commit messages standard, but this it's the most important: https://gist.github.com/tonibardina/9290fbc7d605b4f86919426e614fe692#subject-line-standard-terminology

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+document.getElementById('addItem').addEventListener('click', () => addItem());
```
Comment (201flaviosilva) [inline src/index.js:1]: It's not worrying, and it even depends on each person's taste, but in this case you are passing an arrow function as a callback that calls a function, you can simply pass the function as a callback:

```suggestion
document.getElementById('addItem').addEventListener('click', addItem);
```

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+function addItem() {
+	const newLi = document.createElement('li');
+	const newP = document.createElement('p');
+	const newButton = document.createElement('button');
+	const list = document.getElementById('list');
```
Comment (201flaviosilva) [inline src/index.js:8]: This should be at the start of the document, you don't want to get the element every time that you are trying to add an item.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	const input = document.getElementById('userInput').value;
+
+	newP.innerHTML = input;
+	newButton.innerHTML = 'x';
+
+	document.getElementById('userInput').value = '';
```
Comment (201flaviosilva) [inline src/index.js:14]: In [line 9](https://github.com/pedrosilva410/todolist/pull/4/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R9), you already had the element

**Review by 201flaviosilva — APPROVED** [review]
It's ok to go :)

File: src/index.js (hunk 1 above)
Comment (pedrosilva410) [reply src/index.js:1]: yea that makes more sense

File: src/index.js (hunk 2 above)
Comment (pedrosilva410) [reply src/index.js:8]: yup

File: src/index.js (hunk 3 above)
Comment (pedrosilva410) [reply src/index.js:14]: got it

**Pull request merged**